src/ui/BtnAdd/component.tsx

```diff
@@ -0,0 +1,7 @@
+function BtnAdd() {
+  return (
+    <button type="button">Enviar</button>
```

Comment:
let's allow this component to receive children

```tsx
type BtnAddProps = React.PropsWithChildren<{
  className: string;
}>;

function BtnAdd({ children, className }: BtnAddProps) {
  return (
    <button type="button" className={className}>{children}</button>
```

Reply:
ok!!!